engine/engine.go

```diff
@@ -0,0 +1,45 @@
+package engine
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/opsfactory/kappa/config"
+	"github.com/opsfactory/kappa/container/action"
+	"github.com/opsfactory/kappa/container/backend"
+	"github.com/opsfactory/kappa/container/event"
+)
+
+type Engine struct {
+	cfg     config.Config
```

Comment:
@fntlnz and @jnardiello would you use pointers or values?!
I'm using values since right now I don't want the backend or configuration to be externally modified. But maybe I'm missing some use-cases.